resources/aws/environment.go

```diff
@@ -62,9 +62,16 @@ type Environment struct {
 	randomSubnets pulumi.StringArrayOutput
 }
 
+type CloudProviderEnvironment struct{}
+
+func (e *CloudProviderEnvironment) GetInternalRegistry() string {
+	return "669783387624.dkr.ecr.us-east-1.amazonaws.com"
+}
+
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
I'd rather have `aws.Environment` implementing the interface

```suggestion
var _ CloudProviderEnvironment = &Environment{}

func (e *Environment) GetInternalRegistry() string {
	return "669783387624.dkr.ecr.us-east-1.amazonaws.com"
}
```

Reply:
I am not sure understand how I should use it if I do this. Because I wanted to have 
an instance of `CloudProviderEnvironment` in `CommonConfig`.
Then I would have `aws.Environment` has an attribute for `CommonConfig` and `CommonConfig` that has an attribute pointing to `aws.Environment

Reply:
Modified the code!